### DISCUSSION TITLE
How to combine watchDebounced and watchIgnorable?

### DISCUSSION THREAD

**Discussion by iBobik** [vueuse/vueuse#2414] [Q&A] [answered]
Hello,
Is it possible to watch variable with possibility to not trigger it when callback changes watched variable (watchIgnorable) and also filter it to not trigger too many times in a short time frame (watchDebounced)?

**Comment by EShebunin** [comment, marked as answer]
Hi, 
https://vueuse.org/shared/watchWithFilter/#watchwithfilter
`
    const count = ref(0);
    const {ignoreUpdates} = watchIgnorable(count, (v) => {
        console.log(v);
    }, {
        eventFilter: debounceFilter(500),
    });
    const ignoreUpdatesCount = ignoreUpdates(() => count.value++);
`

**Reply by iBobik** [reply]
Thank you. Haven't noticed ignorable supports filter.
